### PR TITLE
CSS の absolute-size および relative-size の記事を更新

### PR DESCRIPTION
**Pull request by mfuji09**

### CODE REVIEW

File: files/ja/web/css/absolute-size/index.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+  font-size: xx-large;
+}
+.xxx-large {
+  font-size: xxx-large;
+}
+```
+
+#### 結果
+
+{{EmbedLiveSample('Comparing the keyword values', '100%', 400)}}
```
Comment (yin1999) [inline files/ja/web/css/absolute-size/index.md:122]:
```suggestion
{{EmbedLiveSample('キーワード値の比較', '100%', 400)}}
```

File: files/ja/web/css/relative-size/index.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+  font-size: smaller;
+}
+.larger {
+  font-size: larger;
+}
+```
+
+#### 結果
+
+{{EmbedLiveSample('Comparing the keyword values', '100%', 100)}}
```
Comment (yin1999) [inline files/ja/web/css/relative-size/index.md:60]:
```suggestion
{{EmbedLiveSample('キーワード値の比較', '100%', 100)}}
```

**Review by yin1999 — COMMENTED** [review]
It would be a good idea to use the correct id. Or we may facing the problem like this: #16723.

**Comment by mfuji09** [comment]
@yin1999
Thank you!

**Pull request merged**